build.sh

```
@@ -360,6 +382,9 @@ OPTIONS:
     --cpp, -c       Build only C++ static library
     --cpp-test      Build C++ test executable (requires --cpp first)
     --go, -g        Build only Go server (requires C++ library to be built)
+    --test, -t      Run Go unit tests (sets up CGO env for office_oxide).
+                    Pass extra args after `--` to forward to `go test`, e.g.
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)